packages/@react-native-windows/cli/src/config/configUtils.ts

```diff
@@ -164,6 +178,14 @@ export function findDependencyProjectFiles(winFolder: string): string[] {
 function isRnwAppProject(filePath: string): boolean {
   const projectContents = readProjectFile(filePath);
 
+  const rnProjectType = getReactNativeProjectType(
+    tryFindPropertyValue(projectContents, 'ReactNativeProjectType'),
+  );
+
+  if (rnProjectType !== 'unknown') {
+    return true;
+  }
```

Comment:
This is a little bit yucky, since the CLI's MSBuild property evaluation is not reliable. Ideally we should be avoiding adding our own MSBuild props specific to the CLI. I think it might not be complete. E.g. I'm imagining there would be an `App-Uwp` or `Lib-Reunion`? 

Is it possible to hold off this part of the change until it is more developed?

Reply:
I'm less familiar with these changes, @asklar?

Reply:
could removing this and updating the `of (projectLang === 'cs')` check to also return true if we're importing some props file(that we're importing in WinAppSDK app project file) accomplish the same behavior?

Reply:
I like that solution 👍

Reply:
addressed.